cmp/leftRightChooser/impl/ItemRenderer.js

```diff
@@ -4,15 +4,26 @@
  *
  * Copyright © 2018 Extremely Heavy Industries Inc.
  */
+import {PropTypes as PT} from 'prop-types';
 import {Component} from 'react';
 import {div} from 'hoist/layout';
 import {Icon} from 'hoist/icon';
 
 /**
  * Render items in the LeftRightChooser.
+ *
+ * This is an implementation class private to Hoist
+ * @private
  */
 export class ItemRenderer extends Component {
 
+    static propTypes = {
```

Comment:
Not sure we need this, the props here are handled by ag-grid. Don't think it hurts anything either.

Reply:
Would not include as it's an impl class and I think *not* having the propTypes is a useful signal to that effect.